### PR TITLE
REL: release tools derived from Flask

### PR DESCRIPTION
**Pull request by wasade**
Tools for cutting a release and make it all simple like

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # moi changelog
 
-## Version 0.1.0-dev, changes go here
+## Version 0.1.0-dev
+
+Currently in development, released on May 2nd 1023
```
Comment (adamrp) [inline CHANGELOG.md]: Perhaps the most important event that year besides [The Ghaznavid Empire's occupation of Transoxiana](http://en.wikipedia.org/wiki/1023#Asia)

Comment (wasade) [inline CHANGELOG.md]: hah... oops, ya, had that in when prodding the tool for testing. not
convinced it fully works yet, will resolve last kinks when releasing 0.2

Comment (wasade) [inline CHANGELOG.md]: ...good 'ole Transoxiana...

**Comment by josenavas** [comment]
:+1:

**Comment by wasade** [comment]
Once #33 is in, I think this can be merged. Though, I'll likely be making some final tweaks before the release

**Comment by josenavas** [comment]
@wasade I think everything else have been merged. Do you want this to be merged now or you prefer some tests before?

**Comment by wasade** [comment]
Let's merge, and I'll resolve the remaining kinks to cut the release

**Comment by josenavas** [comment]
Great! Done. Let me know if you need anything else to ve reviewed :)

**Comment by wasade** [comment]
Thanks!

**Pull request merged**